index.js

```diff
@@ -92,7 +92,7 @@ module.exports = {
     }
 
     if (type === 'app-boot') {
-      return fastbootAppModule(config.modulePrefix, JSON.stringify(config.APP || {}));
+      return fastbootAppModule(config.modulePrefix, JSON.stringify(config.APP || {}), this.project.isModuleUnification());
```

Comment:
Hmm, we may need to handle the case where the function doesn’t exist (IIRC it only exists on ember-cli master).

Reply:
as in `this.project.isModuleUnification && this.project.isModuleUnification()` ?

Reply:
or perhaps `(typeof this.project.isModuleUnification === 'function') && this.project.isModuleUnification()`

Reply:
fixed.